### PR TITLE
(feature) Export Performance indicators

### PR DESCRIPTION
**Pull request by Blaumaus**
ASANA Ticket: [Export Performance indicators](https://app.asana.com/0/1201849173362898/1202361721375140/f)

UI Demonstration:
Screenshot from 2022-06-15 18-45-47

### CODE REVIEW

File: src/components/StrategyPerfomanceMetrics/StrategyPerfomanceMetrics.helpers.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    [t('strategyEditor.largestGain')]: maxPL,
+    [t('strategyEditor.largestLoss')]: minPL,
+    [t('strategyEditor.drawdown')]: drawdown,
+    [t('strategyEditor.retPerc')]: retPerc,
+    [t('strategyEditor.equityCurve')]: equityCurve,
+    [t('strategyEditor.ret')]: ret,
+    [t('strategyEditor.allocation')]: allocation,
+    [t('strategyEditor.positionSize')]: positionSize,
+    [t('strategyEditor.currentAllocation')]: currentAllocation,
+    [t('strategyEditor.availableFunds')]: availableFunds,
```
Comment (bhoomij) [inline src/components/StrategyPerfomanceMetrics/StrategyPerfomanceMetrics.helpers.js:56]: I'm not sure, but is it possible to define common helper to retrieve these label/values for both rendering/export functions, so code becomes easy to maintain ?

Comment (Blaumaus) [reply src/components/StrategyPerfomanceMetrics/StrategyPerfomanceMetrics.helpers.js:56]: Yes, it should be possible. Working on that.

**Comment by Blaumaus** [comment]
image

File: src/components/BacktestTradesTable/BacktestTradesTable.helpers.js [hunk 2]
```diff
@@ -4,12 +4,7 @@ import _map from 'lodash/map'
 import _split from 'lodash/split'
```
Comment (bhoomij) [inline src/components/BacktestTradesTable/BacktestTradesTable.helpers.js:4]:
```suggestion
```

**Review by bhoomij — APPROVED** [review]

**Pull request merged**